structs.go

```diff
@@ -14,38 +14,38 @@ type BotOrder struct {
 
 //TradeEventData - container for bot trading new event data
 type TradeEventData struct {
-	OrderID        int64
-	OrderAwaitQty  float64 //initial order qty
-	OrderFilledQty float64 //event executed qty
-	Status         string  //used in bot.getOrderData
+	OrderID        int64   `json:"orderID"`
+	OrderAwaitQty  float64 `json:"await_qty"`  //initial order qty
+	OrderFilledQty float64 `json:"filled_qty"` //event executed qty
```

Comment:
Please, use the same naming style. In our case the most appropriate is `camelCase`